src/views/faq/l10n.json

```diff
@@ -71,109 +81,118 @@
     "faq.remixableTitle":"Why does the Scratch Team require that all projects be “remixable”?",
     "faq.remixableBody":"We believe that remixing other people’s projects is a great way to learn to program and to create interesting projects. Through remixing, creative ideas spread through the Scratch community, and everyone benefits. All projects shared on the Scratch website are covered by the “Creative Commons Share Alike” license, which means that you can remix any project you see on the Scratch website -- and everyone else can remix any of the projects that you share on the website.",
     "faq.creativeCommonsTitle":"What if I don’t want others to remix my projects?",
     "faq.creativeCommonsBody":"Remixing is an important part of the Scratch community. If you don’t want others to view or remix your creations, you can still create projects on the Scratch website, but don’t share them on the website.",
     "faq.fairUseTitle":"Can I use images / sounds / media from the internet in my projects?",
-    "faq.fairUseBody":"If you choose to integrate someone else’s work into your own, be sure to give them credit on the project “credits” section, and include a link back to the original. To find art / sounds that are already licensed for remixing, check out the <a href=\"http://search.creativecommons.org/\">Creative Commons search page</a>.",
+    "faq.fairUseBody":"If you choose to integrate someone else’s work into your own, be sure to give them credit on the project “credits” section, and include a link back to the original. To find art / sounds that are already licensed for remixing, check out the {ccLink}.",
+        "faq.ccLinkText":"Creative Commons search page",
     "faq.whyAccountTitle":"Why is it useful to have a Scratch account?",
     "faq.whyAccountBody":"Even without an account, you can play other people’s projects, read comments and forums, and even create your own projects. But you need an account to save and share projects, write comments and forum posts, and participate in other \"social\" activities in the community (like \"loving\" other people’s projects).",
     "faq.createAccountTitle":"How can I create an account?",
-    "faq.createAccountBody":"Just click <i>Join</i> on the Scratch home page. You’ll need to respond to a few questions, and provide an email address. It takes just a couple minutes, and it’s totally free!",
+    "faq.createAccountBody":"Just click Join on the Scratch home page. You’ll need to respond to a few questions, and provide an email address. It takes just a couple minutes, and it’s totally free!",
```

Comment:
We have a lot of places where we used `<i>` to indicate a command or button. I think it would be good to keep them, but then we need a new placeholder convention for it so that it's easy for translators to know what's going on. How about` {iJoin}`? Looking at other uses, there would be `{iAccountSettings}`, `{iExtensions}` etc. The weird one would be the `{iDeleteAccount}` because it doesn't really make sense to use the whole string. If you have a better idea for naming, please try it!

Reply:
What about `Just click \"Join\" on the`.... ?

Reply:
Using italics isn't the normal way to indicate a command or button. More frequently, people use quotes (i.e. click "Join" on the Scratch home page) or (at least in English) simply put it in title case and describe it as a button (i.e. click the Join button on the Scratch home page). Italics is awkward and wastes a lot of overhead, file size, and time. I think (retrospectively) that quotes best delimit a button, especially when one is told to "click" it. I'll switch to quotes.